resources/views/contacts.blade.php

```diff
@@ -0,0 +1,100 @@
+<x-app-layout :page-title="__('Contact Us')">
+    <div class="lg:grid grid-cols-2 gap-5">
+        <div>
+            <x-section>
+                <h2>Admins and Moderators</h2>
+
+                <p class="mb-3">
+                    <a href='/createmessage.php?t=RAdmin'>Send a message to RAdmin</a> for:
+                    <ul>
+                        <li>&#x2022; Reporting offensive behavior.</li>
+                        <li>&#x2022; Reporting copyrighted material.</li>
+                        <li>&#x2022; Reporting cheating to be investigated.</li>
+                        <li>&#x2022; Requesting to be untracked.</li>
+                    </ul>
+                </p>
+            </x-section>
+
+            <x-section>
+                <h2>Developer Compliance</h2>
+
+                <p class="mb-3">
+                    <a href='/createmessage.php?t=DevCompliance'>Send a message to DevCompliance</a> for:
+                    <ul>
+                        <li>&#x2022; Requesting set approval or early set release.</li>
+                        <li>&#x2022; Reporting achievements or sets with unwelcome concepts.</li>
+                        <li>&#x2022; Reporting sets failing to cover basic progression.</li>
+                    </ul>
+                </p>
+            </x-section>
+
+            <x-section>
+                <h2>Quality Assurance</h2>
+
+                <p class="mb-3">
+                    <a href='/createmessage.php?t=QATeam'>Send a message to QATeam</a> for:
+                    <ul>
+                        <li>&#x2022; Reporting a broken set, leaderboard, or rich presence.</li>
+                        <li>&#x2022; Reporting achievements with grammatical mistakes.</li>
+                        <li>&#x2022; Requesting a set be playtested.</li>
+                        <li>&#x2022; Hash compatibility questions.</li>
+                        <li>&#x2022; Hub organizational questions.</li>
+                        <li>&#x2022; Getting involved in a QA sub-team.</li>
+                    </ul>
```

Comment:
nit: Embedding unicode `&#x2022;` characters on each list item should be achievable automatically with CSS & Tailwind at the UL level.

```html
<ul class="list-disc marker:text-[color:var(--variable-name)]>
```

Reply:
> I was surprised it was necessary to add those in the first place.

Yeah Tailwind's built-in reset does this automatically; not sure if it's being applied or not.

> Your suggestion puts the bullets in the margin

Oops - I forgot that is the default behavior of bullets inside a UL. I bet this fixes it:

```html
<ul class="list-disc list-inside"> // probably don't need the marker declaration
</ul>
```